manifests/init.pp

```diff
@@ -10,14 +10,16 @@
   Boolean $debug = false,
   Boolean $trace = false,
   Boolean $announce_cluster = false,
-  String $binpath = "/usr/sbin/gnatsd",
-  String $configdir = "/etc/gnatsd",
-  String $piddir = "/var/run",
+  Stdlib::Absolutepath $binpath = "/usr/sbin/gnatsd",
```

Comment:
Hmm I must have been blind then sorry. Still only on my phone while moving hehe GitHub mobile not that awesome. Sorry